blocks/sheet/index.js

```diff
@@ -51,7 +51,7 @@ async function getData(url) {
   const names = json[':names'];
 
   // Single sheet
-  if (json.data) {
+  if (json[':type'] === 'sheet') {
```

Comment:
There a test for this?

Reply:
There are no unit tests yet for any of this area, but I'll add a test for this change.